src/e2e-test/regression-test/util/interceptors.js

```diff
@@ -35,17 +35,30 @@ class ExtraDynamicFieldInterceptor extends Interceptor {
     }
 
     __handle (data) {
-        if (!('fhirResource' in data && 'entry' in data['fhirResource'])) {
-            return data;
-        }
-        const entries = data['fhirResource']['entry'];
+        const entries = data['entry'];
         if (!_.isArray(entries)) {
             return data;
         }
         for (const entry of entries) {
             if ('resource' in entry && 'resourceType' in entry['resource']) {
                 if (entry['resource']['resourceType'] === 'DocumentReference') {
-                    entry['resource']['date'] = 'removed';
+                    const resource = entry['resource'];
+                    resource['date'] = 'removed';
+
+                    // The zlib.gzip result will be different on different platforms, see https://stackoverflow.com/questions/26516369/zlib-gzip-produces-different-results-for-same-input-on-different-oses.
+                    // Hence the hash result will be different too, which will trigger NodeJS CI error.
+                    if ('content' in resource && _.isArray(resource['content'])) {
```

Comment:
Shall we keep consistent conversion result in different platforms?

Reply:
Currently the difference will only impact regression test, and it will not it doesn't affect function(since the result encoded on Ubuntu can be correctly decoded on Windows).